content/applications/websites/ecommerce/products.rst

```diff
@@ -4,266 +4,341 @@
+     :doc:`SEO <../../websites/website/pages/seo>`.
+   - To check the language(s) of your website, go to :menuselection:`Website --> Configuration -->
+     Settings --> Website Info section`.
 
 Website availability
 --------------------
 
-A product can be set available on either *one* or *all* websites, but it is not possible to select
-*some* websites and not others. To define a product's availability, go to :menuselection:`Website
---> eCommerce --> Products`, select your product, and in the :guilabel:`Sales` tab, click the
-:guilabel:`Website` you wish the product to be available on. Leave the field empty for the products
-to be available on *all* websites.
+A product can be available on either *one* website or *all* websites. It is not possible to select
+*some* websites and not others.
+
+To set availability, open the :ref:`product form <products/product-form>`, go to the
+:guilabel:`Sales` tab, and in the :guilabel:`eCommerce shop` section, and select the
+:guilabel:`Website` you wish the product to be available on. Leave the field blank to make the
+product available on *all* websites.
+
+.. _products/stock-management:
 
 Stock management
 ================
 
-Under the :menuselection:`Website --> Configuration --> Settings --> Shop - Products`, you can
-enable and configure inventory management options.
+Go to :menuselection:`Website --> Configuration --> Settings --> Shop - Products`, to enable and
+configure inventory management options.
 
 .. important::
-   To display the stock level on the product page, the :guilabel:`Product Type` on the **product's
-   form** must be set to :guilabel:`Storable` (only available when the **Inventory** app is
-   installed).
+   - The **Inventory** app must be installed to see the inventory management options.
+   - To display the stock level on your product page, the :guilabel:`Product Type` must be set to
+     :guilabel:`Storable` in the :ref:`product form <products/product-form>`.
 
 Inventory
 ---------
 
-In the :guilabel:`Inventory Defaults` sub-section, you can select the eCommerce selling strategy of
-products:
-
-- :guilabel:`Warehouse`: if you have multiple warehouses, you can define the warehouse associated to
-  your website. If you have multiple websites, you can select a different one per website;
-- :guilabel:`Out-of-Stock (Continue Selling)`: enabling it allows customers to continue placing
-  orders even when the product is **out-of-stock**.
-  Leave it  unchecked to **prevent orders**;
-- :guilabel:`Show Available Qty`: enabling it displays the available quantity left under a specified
-  threshold on the product page. The available quantity is calculated based on the 'On hand'
-  quantity minus the quantity already reserved for outgoing transfers.
-
-.. seealso::
-   :ref:`Allow only selected customers to buy <cart/prevent-sale>`
+In the :guilabel:`Inventory Defaults` sub-section, select your eCommerce selling strategy:
```

Comment:
```suggestion
Set the following fields:
```
I'm not sure they all relate to the selling strategy per say so I would try to stay a little more general (e.g., the Warehouse field, which is missing from the list of field btw)

Reply:
Updated, as discussed.